samples/v1.6/Elements/Carousel.ScenarioCards.Timer.json

```diff
@@ -2,463 +2,465 @@
+	    											"title": "High",
+	    											"value": "50"
+	    										},
+	    										{
+	    											"title": "Low",
+	    											"value": "32"
+	    										}
+	    									]
+	    								}
+	    							],
+	    							"selectAction": {
+	    								"type": "Action.OpenUrl",
+	    								"title": "View Saturday",
+	    								"url": "https://www.microsoft.com"
+	    							}
+	    						}
+	    					]
+	    				}
+	    			]
+	    		},
+	    		{
+	    			"type": "CarouselPage",
+	    			"id": "sportingEvent",
+	    			"selectAction": {
+	    				"type": "Action.OpenUrl",
+	    				"title": "Click for more information about the first carousel page!",
+	    				"url": "https://adaptivecards.io/"
+	    			},
+	    			"items": [
+	    				{
+	    					"type": "ColumnSet",
+	    					"columns": [
+	    						{
+	    							"type": "Column",
+	    							"width": "auto",
+	    							"items": [
+	    								{
+	    									"type": "Image",
+	    									"url": "https://adaptivecards.io/content/cats/3.png",
+	    									"size": "medium",
+	    									"altText": "Shades cat team emblem"
+	    								},
+	    								{
+	    									"type": "TextBlock",
+	    									"text": "SHADES",
+	    									"horizontalAlignment": "center",
+	    									"weight": "bolder",
+	    									"wrap": true
+	    								}
+	    							]
+	    						},
+	    						{
+	    							"type": "Column",
+	    							"width": "stretch",
+	    							"separator": true,
+	    							"spacing": "medium",
+	    							"items": [
+	    								{
+	    									"type": "TextBlock",
+	    									"text": "Sat, Aug 31, 2019",
+	    									"horizontalAlignment": "center",
+	    									"wrap": true
+	    								},
+	    								{
+	    									"type": "TextBlock",
+	    									"text": "Final",
+	    									"spacing": "none",
+	    									"horizontalAlignment": "center",
+	    									"wrap": true
+	    								},
+	    								{
+	    									"type": "TextBlock",
+	    									"text": "45 - 7",
+	    									"size": "extraLarge",
+	    									"horizontalAlignment": "center",
+	    									"wrap": true
+	    								}
+	    							]
+	    						},
+	    						{
+	    							"type": "Column",
+	    							"width": "auto",
+	    							"separator": true,
+	    							"spacing": "medium",
+	    							"items": [
+	    								{
+	    									"type": "Image",
+	    									"url": "https://adaptivecards.io/content/cats/2.png",
+	    									"size": "medium",
+	    									"horizontalAlignment": "center",
+	    									"altText": "Skins cat team emblem"
+	    								},
+	    								{
+	    									"type": "TextBlock",
+	    									"text": "SKINS",
+	    									"horizontalAlignment": "center",
+	    									"weight": "bolder",
+	    									"wrap": true
+	    								}
+	    							]
+	    						}
+	    					]
+	    				}
+	    			]
+	    		},
+	    		{
+	    			"type": "CarouselPage",
+	    			"id": "stockUpdate",
```

Comment:
looks like some spaces snuck in here -- probably worth running `prettier` on all everything under `samples/v1.6`

Reply:
I updated the formatting on all of those files